ui/pages/token-allowance/token-allowance.js

```diff
@@ -263,13 +264,13 @@ export default function TokenAllowance({
     );
   };
 
-  const handleNextNonce = () => {
+  const handleNextNonce = useCallback(() => {
     dispatch(getNextNonce());
-  };
+  }, [getNextNonce, dispatch]);
 
   useEffect(() => {
     handleNextNonce();
-  }, [dispatch]);
+  }, [handleNextNonce]);
```

Comment:
handleNextNonce is used somewhere else. But we also want to getNextNonce once the page loads. I remember there was an issue with nonce that necessitated this change.

I've however changed the useEffect to call getNextNonce instead of handleNextNonce